### PR TITLE
Add more vaccinator autoshoot checks and ESP details

### PR DESCRIPTION
**Pull request by surepy**
just a minor thing that bugged me before

### CODE REVIEW

**Comment by LightPower392** [comment]
I'd like to see more weapons included. Also, you can look up a weapon on the tf2 wiki and search for its damage type.

**Comment by surepy** [comment]
> I'd like to see more weapons included. Also, you can look up a weapon on the tf2 wiki and search for its damage type.

added Pomson 6000; I don't know what else I am missing.

for The Loose Cannon (CTFCannon) I'm having trouble deciding if I should add or not.
because the initial knockback damage does not count as a blast, pretty sure.

**Comment by LightPower392** [comment]
I meant adding projectiles like the direct hit and the grenade launcher.

**Comment by surepy** [comment]
> I meant adding projectiles like the direct hit 

```
case CL_CLASS(CTFRocketLauncher_DirectHit):
```
> and the grenade launcher.

```
case CL_CLASS(CTFGrenadeLauncher):
```

it's there tho

File: src/hacks/ESP.cpp [hunk 1]
```diff
@@ -1207,13 +1207,30 @@ void _FASTCALL ProcessEntity(CachedEntity *ent)
                     if (IsPlayerInvulnerable(ent))
                         AddEntityString(ent, "*INVULNERABLE*");
                     // Vaccinator
+                    // Vaccinator
```
Comment (BenCat07) [inline src/hacks/ESP.cpp:1210]: comment

**Comment by TotallyNotElite** [comment]
Let me check something real quick.

**Comment by surepy** [comment]
note to self: check if system time is always correct so time travel on commits don't happen

**Comment by LightPower392** [comment]
lol. I think this looks good

**Pull request merged**